### DISCUSSION TITLE
Is there a way to force sharp corners with field margin?

### DISCUSSION THREAD

**Discussion by Samus1111111** [Courseplay/Courseplay_FS22#1722] [Q&A] [answered]
So kind of a follow-on to a previous question I had: https://github.com/Courseplay/Courseplay_FS22/discussions/1611

I'm using a negative field margin because some of the corners of my fields don't have sharp corners (even when I set Courseplay to use sharp corners). However, I've found that as I use negative field margins, it actually makes previously sharp corners into smooth corners. These two facts seem to be fighting against each other. The generated courses are missing corners, so I use a field margin, but this causes the courses to use smooth corners, which then causes those corners to be missed again even with maximum headland overlap.

Is there any way to force courseplay to use sharp corners, so that I don't have missed areas, regardless of the field margin I'm using? A manual override, if you will.

**Comment by pvaiko** [comment, marked as answer]
Sorry, no such thing is implemented yet, we have #688 open for this already.
